### PR TITLE
Mock search

### PR DESCRIPTION
**Pull request by alyxrossetti**

### CODE REVIEW

File: gpi/frontend/web/src/ui/components/getty-type-ahead.vue [hunk 1]
```diff
@@ -0,0 +1,327 @@
...
+
+    getResponse: {
+      required: false,
+      type: Function,
+      default: (response) => {
+        return response.data.result
+      },
+    },
+
+    fetch: {
```
Comment (rit) [inline gpi/frontend/web/src/ui/components/getty-type-ahead.vue]: @alyxrossetti `axios` should be used in the Vuex's actions. This component should use Vuex's getter to access (wait for) the data set by the Vuex's action.

File: gpi/frontend/web/src/ui/components/getty-type-ahead.vue [hunk 2]
```diff
@@ -0,0 +1,327 @@
...
+      required: false,
+      type: Array,
+      default () {
+        this.items
+      },
+    },
+  },
+  data () {
+    return {
+      items: [],
```
Comment (rit) [inline gpi/frontend/web/src/ui/components/getty-type-ahead.vue]: it looks like `items` should be a custom list/array. We can make use this package: https://github.com/sindresorhus/cycled

With the change from list to "cycled list", the `up` and `down` methods should be much cleaner.

File: gpi/frontend/web/src/ui/components/getty-type-ahead.vue (hunk 1 above)
Comment (alyxrossetti) [reply gpi/frontend/web/src/ui/components/getty-type-ahead.vue]: @rit this is done

**Comment by alyxrossetti** [comment]
@rit @workergnome I think this is ready for review.

**Pull request merged**